atqa-framework/browsers.json

```diff
@@ -0,0 +1,55 @@
+{
+  "chrome": {
+    "default": "102.0",
+    "versions": {
+      "102.0": {
+        "image": "selenoid/vnc_chrome:102.0",
+        "port": "4444",
+        "path": "/",
+        "screenResolution": "1920x1080x24",
+        "volumes": [
+          "/home/runner/work/waves-keeper/waves-keeper/atqa-framework/extension:/home/selenium"
```

Comment:
So it wouldn't work locally with these paths, right? Is there a way to avoid hard-coding them?

Reply:
Yes, it does. As you can see this one is json, so the only way to run it locally use a separate config. It has been added to gitignore as _browser.bckp.json_ so u have to create this one and declare your own paths there.

Reply:
Copying and patching config doesn't seem like a good way to have it running locally. Where is this documented?

Reply:
https://aerokube.com/selenoid/latest/